### PR TITLE
refactor: link tests, small code refactor and amend readme

### PR DESCRIPTION
**Pull request by colincclark**

### CODE REVIEW

File: packages/link/README.md [hunk 1]
```diff
@@ -19,7 +20,44 @@ import Link, { TextLink } from "@times-components/link";
...
+```
+yarn test:all
+yarn test:android
+yarn test:ios
+yarn test:web
+```
+
+Visit the official
+[storybook](http://components.thetimes.co.uk/?knob-Size%20of%20ad%20placeholder%3A=default&selectedKind=Primitives%2FLink&selectedStory=Link%20with%20big%20content&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybooks%2Fstorybook-addon-knobs)
+to see our available link templates.
```
Comment (aronshamash) [inline packages/link/README.md]: I don't think we refer to the storybook stories as 'templates'? What about 'to see examples of `link` in use'?

Comment (aronshamash) [reply packages/link/README.md]: As discuss this is not in scope.

File: packages/link/README.md [hunk 2]
```diff
@@ -1,7 +1,8 @@
-# Link Component
+# Link
 
-This component wraps its children to make them clickable and turn them into
-hyperlinks for web.
+The link component wraps its children and makes them clickable (web) or
```
Comment (craigbilner) [inline packages/link/README.md]: perhaps a bit more around the difference between them, one is to wrap anything whereas on is specifically for an inline linked word etc?

minor: would be good to use backticks to differentiate code/names from the rest of the doc

Comment (colincclark) [reply packages/link/README.md]: done

**Pull request merged**